src/SprykerEco/Shared/Heidelpay/Transfer/heidelpay.transfer.xml

```diff
@@ -18,6 +20,7 @@
         <property name="amortisationText" type="string" />
         <property name="accruingInterest" type="float" />
         <property name="totalAmount" type="float" />
+        <property name="privacyPolicyStatus" type="bool" />
```

Comment:
`Status` doesn't look like bool value to me

Reply:
agree, changed to `isPolicyAgreementChecked`